openmindat/countries.py

```diff
@@ -0,0 +1,206 @@
+from . import mindat_api
+
+
+class CountriesRetriever:
```

Comment:
Second revision:
1. The endpoint to this class is `countries_list`.  Therefore, please consider using another class name, e.g., "CountriesListRetriever".
2. The querying result for this class is incomplete: it only returns one page of items. Please consider programming a specific function for obtaining the countries in a for loop using your new function of getting the list results. This would be a tentative solution for users to obtain the entire countries list.